tr.org.pardus.pen.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: tr.org.pardus.pen
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+command: /app/usr/bin/pardus-pen
```

Comment:
```suggestion
command: pardus-pen
```

Reply:
if it install the binary in `/app/usr/bin` then it's just wrong.